### PR TITLE
build: improving docker-compose configuration

### PR DESCRIPTION
**Pull request by alexgarel**
Some improvement in the docker-compose configuration.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ For development, you have two options for running the service:
 
 To develop on docker, make the changes you need, then build the image and compose by running:
 ```console
-docker build -t off_search_image . && docker-compose up -d
+docker-compose up -d
 ```
 
 However, this tends to be slower than developing locally.
```
Comment (alexgarel) [inline README.md:37]: I don't know if you where aware you don't have to rebuild if you mount the code dynamically (as done in dev.yml)

File: .env [hunk 2]
```diff
@@ -7,8 +14,13 @@ CLUSTER_NAME=docker-cluster
 # Set to 'basic' or 'trial' to automatically start the 30-day trial
 LICENSE=basic
 
-# Port to expose Elasticsearch HTTP API to the host
+# Exposing some services to the host
 ES_PORT=127.0.0.1:9200
+ES_VUE_PORT=127.0.0.1:8080
+SEARCH_PORT=127.0.0.1:8000
```
Comment (alexgarel) [inline .env:20]: I prefer 
- all port exposure to be configurable (to be able to launch more than one project if needed)
- default to exposure on localhost for dev (for the case where you might be on a public network, like in the train…).

**Comment by alexgarel** [comment]
@simonj2, I was about to comment, but I think it's explicit enough, do not hesitate to ask questions if needed.

File: README.md (hunk 1 above)
Comment (simonj2) [reply README.md:37]: Haven't used that before! Would you mind removing this one and replacing the line above with perhaps:
The service on docker is mounted to your local app directory, so changes will take effect automatically. For changes to the other infrastructure, you will need to re-run `docker-compose`.

**Review by simonj2 — APPROVED** [review]
This is much better - thank you for doing this!

**Comment by simonj2** [comment]
@alexgarel - Feel free to merge - happy to sync and deploy together, etc

**Comment by teolemon** [comment]
@alexgarel @simonj2 I'm merging

**Pull request merged**